### PR TITLE
[Federalist application] - Set report template to use a sub branch of uswds-federalist template

### PR DESCRIPTION
**Pull request by line47**
Trying this out to set the branch to `report-config` from the https://github.com/18F/federalist-uswds-template/tree/report-config repo. 

More info - https://github.com/18F/federalist/issues/1242#issuecomment-352091592

### CODE REVIEW

**Comment by wslack** [comment]
@line47 can we do this as a new template in the list instead of changing the existing listing?

**Comment by line47** [comment]
Certainly - update the PR incoming

File: config/templates.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
 module.exports = {
   report: {
     owner: '18f',
-    repo: 'federalist-report-template',
+    repo: 'federalist-uswds-template',
```
Comment (jseppi) [inline config/templates.js]: was this change intentional, @line47?

Comment (line47) [reply config/templates.js]: Nope, thanks for catching 👍

**Comment by line47** [comment]
🎉 🎉 🎉
